### PR TITLE
Finished Dutch Translation by Frank Beentjes

### PR DESCRIPTION
**Pull request by Fbeen**
Dont forget to add nl in the configuration file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@
 /vendor/
 /bin/
 /composer.phar
+/nbproject/private/
```
Comment (bocharsky-bw) [inline .gitignore]: I think this is a very personal and we should avoid it. BTW, many of devs use PhpStorm instead

**Comment by bocharsky-bw** [comment]
You should revert changes in `web/config.php`.

Also I think we don't need to update `composer.lock` now. Better to update it when new version of Symfony will be avaliable. Anyway, `composer.lock` updates should be in separate PR.

File: .gitignore (hunk 1 above)
Comment (Fbeen) [inline .gitignore]: Thank you for your comment.

**Comment by javiereguiluz** [comment]
@Fbeen thanks for providing these translation fixes and congrats on your first contribution to this project!

**Comment by javiereguiluz** [comment]
Please note that we use a tool that "squashes" commits before merging pull requests. That's why GitHub displays your pull request as "Closed" instead of "Merged". But trust me that your changes were correctly merged :)

**Pull request closed without merging**